tests/foreman/ui/test_subscription.py

```diff
@@ -554,3 +556,77 @@ def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitle
             'This organization is not using Simple Content Access. Entitlement-based subscription '
             'management is deprecated and will be removed in Satellite 6.16.' in sca_alert
         )
+
+
+@pytest.mark.parametrize('setting_update', ['expire_soon_days'], indirect=True)
+def test_positive_check_manifest_validity_notification(
+    target_sat, setting_update, default_org, function_org
+):
+    """Check notification when manifest is going to expire.
+
+    :id: 29ab0e80-25eb-44f0-9294-3a26922c33c5
+
+    :setup:
+        1. create new organization, let's say "Expire manifest org"
+
+    :steps:
+        1. Upload expired manifest in newly created org
+        2. go to Content > Subscriptions page, click on 'Manage Manifest' button.
+        3. Search for message string 'Manifest expired', or 'Your manifest expired'
+        4. delete expired manifest from this org (cleanup part)
+        5. Upload non-expired manifest in default org
+        6. go to Content > Subscription page, click on 'Manage Manifest' button.
+        7. Search for message string 'Manifest expiring soon', or 'Your manifest will expire'
+        8. delete expired manifest from default org (cleanup part)
+
+    :expectedresults:
+        1. 'Manifest expired' message appear on screen
+
+    :BZ: 2075163
+
+    :customerscenario: true
+    """
+    property_name = setting_update.name
+    property_value = 366  # value should be greater than 365 to get expected output message
```

Comment:
Oh, you are assigning it _after_ other testing on line 623.
Okay, then ignore the above and consider to actually use it like 
```
setting_update.value = 366
setting_update = setting_update.update({'value'})
```
or remove it if you wish to go via `session.settings.update(...)`.

I would vote for the first case since I don't see any need for doing the settings change through UI here.

Reply:
`@pytest.mark.parametrize('setting_update', ['expire_soon_days=366'], indirect=True)` This would update settings before test run, might get different alert message for `Manifest expires soon` scenario.
But I would test it locally and update you back.

Reply:
As per last discussion in standup, I would keep same changes, i.e  updating "setting > content > expire soon days" value inside test.
@vsedmik

Reply:
implemented